style/client.css

```diff
@@ -178,17 +178,20 @@ button:disabled {
 }
 */
 .dark .button,
-.dark .tabbar a.button {
+.dark .tabbar a.button,
+.dark .popupmenu button.button {
```

Comment:
Because otherwise the buttons look like in the top picture, based on the '.button, .popupmenu button.button' rule on line 330.